triage-party/base/route.yaml

```diff
@@ -0,0 +1,9 @@
+---
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: triage-party
```

Comment:
Can we get the route secured via `acme-controller` please? Just add the same annotation as you can see here: 

https://github.com/operate-first/apps/blob/8b54f97f86e2f9938a55486aa7267af62d22e5f3/cloudbeaver/overlays/emea/ai4ci/secure-route.yaml#L5-L6

Reply:
Sure, I can move the annotation(s) to the base if you prefer, but I added this in the kustomization within the overlay: the `Route` there contains things that are specific to the smaug cluster, like the host name for the route and this annotation (because smaug does have acme-controller).

Makes sense?